src/test_l1_dsAudio.c

```diff
@@ -10102,10 +8011,6 @@ int test_l1_dsAudio_register ( void )
 	UT_add_test( pSuite, "dsAudioPortTerm_L1_negative" ,test_l1_dsAudio_negative_dsAudioPortTerm );
```

Comment:
Some APIs are specific to source or sink. If the API is not supported, expected to return NOT_SUPPORTED. Split those test cases by adding _source or _sink postfix and modify accordingly. When yaml changes introduced you can filter the register based on the platform

Reply:
you mean split the test cases like 
test_l1_dsAudio_positive_dsGetMS12AudioProfileList_source
test_l1_dsAudio_positive_dsGetMS12AudioProfileList_sink

but wont that be a duplicate work of the next task(enhancement of using yaml) that we are going to do?